jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridPresenter.java

```diff
@@ -86,33 +100,43 @@ protected ListView getListView() {
 
     public abstract void setupAdvancedSearchView();
 
+    public abstract void setupActiveSearchFilters();
```

Comment:
Perhaps a default implementation of `setupActiveSearchFilters()` could only call `setupDefaultActiveSearchFilters()` and let the child presenters decide if it's necessary to override `setupActiveSearchFilters()`